scripts/tests/tdd-files-list.txt

```diff
@@ -1,5 +1,5 @@
-test/integration/<replace>.test.js
-test/integration/<replace>>.test.js
-test/unit/<replace>/<replace>.test.js
-test/unit/<replace>/<replace>.test.js
-test/unit/<replace>/<replace>.test.js
+test/integration/*.test.js
+test/integration/*.test.js
+test/unit/*/*.test.js
+test/unit/*/*.test.js
+test/unit/*/*.test.js
```

Comment:
Can you remove 2, 4 and 5 lines I think it is not required ?

Reply:
They were already present when i cloned the repo.